packages/docs/src/styles/pages/layout.scss

```diff
@@ -21,33 +21,38 @@ body {
   flex: 1 0 auto;
 }
 
+.page-content__content {
+  > h2 {
+    @extend .ds-h2;
+  }
+  > h3 {
+    @extend .ds-h3;
+  }
+  > h4 {
+    @extend .ds-h4;
+  }
+  > h5 {
+    @extend .ds-h5;
+  }
+  > h6 {
+    @extend .ds-h6;
+  }
+}
+
 @media (min-width: $width-lg) {
   .page-content__content {
     // removing margin for first heading on content pages
     & > :nth-child(2) {
       margin-top: 0;
     }
     > h2 {
-      font-size: $font-size-3xl;
       font-weight: normal;
```

Comment:
Should this font weight come from a token as well? In core it looks like the h2s on our pages are the only headings besides 4xl that aren't bold. Not sure if that's intentional. Based on `t.font['weight-bold']` in the theme file, I'd think it'd be bold.

Reply:
Removing the weight setting here to try it out, good call out.